client/blocks/reader-full-post/_content.scss

```diff
@@ -81,27 +81,15 @@
 	.image-wrapper {
 		position: relative;
 
-		.image-border {
-			content: "";
-			position: absolute;
-			pointer-events: none;
-			top: 0;
-			left: 0;
-			bottom: 0;
-			right: 0;
-			box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.1);
-			border-radius: 6px; /* stylelint-disable-line scales/radii */
-			margin: auto;
-			max-width: 100%;
-		}
-
 		figcaption {
 			border-bottom-left-radius: 6px; /* stylelint-disable-line scales/radii */
 			border-bottom-right-radius: 6px; /* stylelint-disable-line scales/radii */
 		}
 	}
 	.image-wrapper img {
+		border-radius: 6px; /* stylelint-disable-line scales/radii */
```

Comment:
Here I'm just maintaining the styles already established.

Reply:
Can we use existing variable for this purpose either `$radius-medium` or `$radius-large`? We don't have a variable for 6px which we can introduce but I think going with existing ones is also fine.

Reply:
Done 👍